tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_types.ts

```diff
@@ -206,6 +206,26 @@ export interface RunState {
   executions: Executions;
 }
 
+// A source-code file.
+export interface SourceFileSpec {
+  host_name: string;
+  file_path: string;
+}
+
+// A specific line of a source-code file, i.e., a stack frame.
+export interface SourceLineSpec extends SourceFileSpec {
```

Comment:
nit: totally optional and up to you, but for naming, wdyt about `SourceLocation` ?

Reply:
I prefer keeping `SourceLineSpec`. Reasons:

1. It's symmetric w.r.t. `SourceFileSpec` above.
2. It's specific about whether it's talking about a line or a range of lines (the former is the case currently), and whether it's talking about a single stack frame or a list of them (again the former is the case.)